http/src/main/java/com/redhat/lightblue/client/http/LightblueHttpClient.java

```diff
@@ -119,21 +127,30 @@ public LightblueResponse metadata(LightblueRequest lightblueRequest) {
     @Override
     public LightblueResponse data(LightblueRequest lightblueRequest) {
         LOGGER.debug("Calling data service with lightblueRequest: " + lightblueRequest.toString());
-        try {
-            return callService(new LightblueHttpDataRequest(lightblueRequest)
-                    .getRestRequest(configuration.getDataServiceURI()));
-        } catch (RuntimeException e) {
-            throw new LightblueHttpClientException("Error sending lightblue request: " + lightblueRequest.getBody(), e);
-        }
+        return callService(new LightblueHttpDataRequest(lightblueRequest)
+                .getRestRequest(configuration.getDataServiceURI()));
```

Comment:
Well,  I think I can change it back how it was and re-throw  the exception in data(LightblueRequest lightblueRequest, Class<T> type)  method, what do you think?